worker.go

```diff
@@ -56,6 +54,14 @@ func (w *Worker) IngestMetrics(ms []*metricpb.Metric) {
 	w.ImportMetricChan <- ms
 }
 
+// WorkerSet is a set of workers that can ingest metrics. WorkerSet is
+// explicitly defined as its own type because it is possible to have multiple
+// WorkerSets via ComputationRoutingConfig
+type WorkerSet struct {
+	Workers                  []*Worker
+	ComputationRoutingConfig *ComputationRoutingConfig
```

Comment:
I hope it's not too lurky to attach ComputationRoutingConfig to WorkerSet - this was useful particularly because we pass around WorkerSet a lot, including into other packages.